### PR TITLE
Initialized next js project for todo proejct

### PR DESCRIPTION
**Pull request by palakgupta2712**
Date: 4th Nov 2024

Developer Name: Palak Gupta (palakgupta2712@gmail.com)

---

## Description

Initialized next js project with basic configuration of tailwind for todo project

### Documentation Updated?

- [ ] Yes
- [x] No

### Under Feature Flag

- [ ] Yes
- [x] No

### Database Changes

- [ ] Yes
- [x] No

### Breaking Changes

- [ ] Yes
- [x] No

### Development Tested?

- [x] Yes
- [ ] No

## Screenshots
image

</details>

## Test Coverage
<details>
<summary>Screenshot 1</summary>

</details>

### CODE REVIEW

**Review by iamitprakash — CHANGES_REQUESTED** [review]
add PR Desc

File: README.md [hunk 1]
```diff
@@ -1,2 +1,36 @@
...
+- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
+- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+
+You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js) - your feedback and contributions are welcome!
+
+## Deploy on Vercel
+
+The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
+
+Check out our [Next.js deployment documentation](https://nextjs.org/docs/app/building-your-application/deploying) for more details.
```
Comment (yesyash) [inline README.md:36]: - remove this

File: app/globals.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+@media (prefers-color-scheme: dark) {
+  :root {
+    --background: #0a0a0a;
+    --foreground: #ededed;
+  }
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+}
```
Comment (yesyash) [inline app/globals.css:19]: - let's use `next/font` to manage fonts.
- [doc link](https://nextjs.org/docs/app/building-your-application/optimizing/fonts#with-tailwind-css)

File: package.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "todo",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
```
Comment (yesyash) [inline package.json:6]: - does this use turbopack by default or should we add the `--turbopack` flag?

File: tailwind.config.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import type { Config } from "tailwindcss";
+
+const config: Config = {
+  content: [
+    "./pages/**/*.{js,ts,jsx,tsx,mdx}",
+    "./components/**/*.{js,ts,jsx,tsx,mdx}",
+    "./app/**/*.{js,ts,jsx,tsx,mdx}",
```
Comment (yesyash) [inline tailwind.config.ts:7]: - will we have all our components & app folder in a common folder like `src` if so can we just add the path to that common folder?

**Review by yesyash — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (palakgupta2712) [reply README.md:36]: Done.

File: app/globals.css (hunk 2 above)
Comment (palakgupta2712) [reply app/globals.css:19]: Fixed.

File: package.json (hunk 3 above)
Comment (palakgupta2712) [reply package.json:6]: Has to be added explicitly. Missed by me. 
Added now. 
Ref doc:- https://nextjs.org/docs/architecture/turbopack

File: tailwind.config.ts (hunk 4 above)
Comment (palakgupta2712) [reply tailwind.config.ts:7]: So currently, this is how our folder structure is, that's why didn't change this.

**Comment by palakgupta2712** [comment]
Hi @yesyash @iamitprakash 
Addresed your comments. Please review it again.

**Review by yesyash — APPROVED** [review]

**Review by iamitprakash — APPROVED** [review]

**Pull request merged**